docs/source/examples/sharing_schemes.md

```diff
@@ -0,0 +1,146 @@
+   - Each language maintains a distinct set of parameters for both encoder and decoder.
+   - No parameter sharing occurs between languages.
+```yaml
+  train_ar-ar:
+    dec_sharing_group:
+    - ar
+    enc_sharing_group:
+    - ar
+```
+- `train_ar-ar`: This denotes the training configuration for the Arabic-to-Arabic language pair.
+- `dec_sharing_group`: Specifies the decoder sharing group, indicating which languages share decoder parameters. In this case, only Arabic (ar) is included, meaning no sharing with other languages for decoding.
+- `enc_sharing_group`: Denotes the encoder sharing group, signifying which languages share encoder parameters. Here, it's also set to only Arabic (ar), indicating no encoder parameter sharing with other languages.
+
+### 2. **Shared Encoder, Separate Decoder:**
+   - Encoder parameters are shared across all languages.
+   - Each language has a separate set of parameters for the decoder.
+```yaml
+  train_ar-ar:
+    dec_sharing_group:
+    - ar
+    enc_sharing_group:
+    - all
+```
+
+### 3. **Separate Encoder, Shared Decoder:**
+   - Each language has a separate set of parameters for the encoder.
+   - Decoder parameters are shared across all languages.
+
+```yaml
+  train_ar-en:
+    dec_sharing_group:
+    - all
+    enc_sharing_group:
+    - ar
+```
+
+### 4. **Fully Shared:**
+   - Both encoder and decoder parameters are shared across all languages.
+   - The entire model is shared among all language pairs.
+```yaml
+  train_ar-ar:
+    dec_sharing_group:
+    - all
+    enc_sharing_group:
+    - all
+```
+
+You can conveniently download the complete configurations using the following command:
+```bash
+wget https://mammoth-share.a3s.fi/configs.tar.gz
+```
+
+These configurations provide a solid foundation for configuring, training, and testing various sharing schemes in the MAMMOTH framework. Ensure to modify the file paths according to your specific compute device configurations. Feel free to experiment and tailor these settings to suit your specific needs.
+
+## Training Modular Systems
+
+
+### 1. **Setup:**
+To initiate the training process for MAMMOTH's modular systems, start by setting up the necessary environment variables:
+
+```bash
+export MAMMOTH=/path/to/mammoth
+export CONFIG=/path/to/configs/config.yaml
+```
+
+#### 2. **Training Command:**
+
+Execute the following command to commence training:
+
+```bash
+srun /path/to/wrapper.sh $MAMMOTH/train.py \
+    -config $CONFIG \
+    -master_ip $SLURMD_NODENAME \
+    -master_port 9969
+```
+
+For the wrapper script, use an example like the one below:
+```bash
+python -u "$@" --node_rank $SLURM_NODEID
+```
+
+This tutorial utilizes SLURM for job scheduling and parallel computing.
+You can tailor the provided commands for your specific needs, adapting them to alternative job scheduling systems or standalone setups.
+Ensure that the `config.yaml` file specifies the desired sharing scheme.
+
+#### 3. **Testing Command:**
```

Comment:
Maybe inference, rather than testing (as we don't produce scores but predictions)?